### ISSUE TITLE
AsciiDoc dictionnary

### ISSUE THREAD

**Issue by noraj** [streetsidesoftware/cspell-dicts#1968]
It would be nice to have a dictionary for AsciiDoc so it would be aware of the context. cf https://github.com/streetsidesoftware/vscode-spell-checker/issues/1973

**Comment by KieranRobson** [comment]
No, I mean what terms and words flag when using the program and what do you want in the dictionary if it was created, like is it substantial or is it just a few words that can be added to a custom word dict in your software?

**Comment by noraj** [comment]
I would be like for markdown (https://github.com/streetsidesoftware/cspell-dicts/tree/main/dictionaries/markdown#readme) but for asciidoc, another markup language. The idea would be to lower the number of false positive when using [vscode-spell-checker](https://github.com/streetsidesoftware/vscode-spell-checker) (which is using cspell). Because currently the spell checker is working for AsciiDoc but unlike for Markdown it is triggering Unknown word for words inside image links, inside code blocks, in document attributes tag, etc. so it's checking the spell like if it was bare text and is not aware of the asciidoc syntax which make the plugin reach the 100 problems very quickly and then stop to check the spell unless you add all the words to the workspace dictionary as a workaround cf. https://github.com/streetsidesoftware/vscode-spell-checker/issues/1973.
